Ginger/GingerCore/ALM/OctaneCore.cs

```diff
@@ -352,27 +563,42 @@ private static ObservableList<ExternalItemFieldBase> AddFieldsValues(ListResult<
                     itemfield.Name = field.Label;
                     itemfield.Mandatory = field.IsRequired;
                     itemfield.SystemFieled = !field.IsUserField;
-                    
+
                     if (itemfield.Mandatory)
                     {
                         itemfield.ToUpdate = true;
                     }
                     itemfield.ItemType = entityType;
                     itemfield.Type = field.FieldType;
-
-                    if (listnodes != null && listnodes.ContainsKey(field.Name) && listnodes[field.Name].Any())
+                    if (field.FieldType.ToLower() == "reference")
                     {
-                        itemfield.PossibleValues = new ObservableList<string>(listnodes[field.Name]);
-                    }
-                    else if (listnodes != null && listnodes.ContainsKey(entityType.ToLower() + "_" + field.Name) && listnodes[entityType.ToLower() + "_" + field.Name].Any())
-                    {
-                        itemfield.PossibleValues = new ObservableList<string>(listnodes[entityType.ToLower() + "_" + field.Name]);
-                    }
-                    else if (phases != null && phases.ContainsKey(field.Name) && phases[field.Name].Any())
-                    {
-                        itemfield.PossibleValues = new ObservableList<string>(phases[field.Name]);
+                        try
+                        {
+                            BaseEntity temp = (BaseEntity)field.GetValue("field_type_data");
+                            itemfield.IsMultiple = temp.GetBooleanValue("multiple").Value;
+                        }
+                        catch { }
```

Comment:
Codacy found an issue: [Handle the exception or explain in a comment why it can be ignored.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6361231)